src/interface/desktop/chat.html

```diff
@@ -888,10 +888,10 @@
         }
 
         function renderFirstRunSetupMessage() {
-            first_run_message = `Hi 👋🏾, to get started:
+            first_run_message = `<p class="inline-chat-heading">Hi 👋🏾, to get started:<p>
```

Comment:
Could we give this a more specific class name, so that it's only applied to the first run message?